src/pip/_internal/models/format_control.py

```diff
@@ -42,7 +39,7 @@ def __repr__(self):
 
     @staticmethod
     def handle_mutual_excludes(value, target, other):
-        # type: (str, Optional[Set[str]], Optional[Set[str]]) -> None
+        # type: (str, Set[str], Set[str]) -> None
```

Comment:
This can be done since `target`, and `other` refer to the `no_binary` and `only_binary` instance attribute of `FormatControl` class, and these values are set to `set()` if not provided in the constructor.